synapse/config/server.py

```diff
@@ -1234,6 +1238,12 @@ def generate_config_section(
           # information about using custom templates.
           #
           #custom_template_directory: /path/to/custom/templates/
+
+        # List of rooms to exclude from sync responses.
```

Comment:
Can some info be added here about the use case for excluding rooms and the end result (i.e. that end-users won't see these rooms in their clients, etc?)